strapi/strapi
# future(homepage): add generic widget component

### What does it do?

Adds a widget component that will be the shell for each homepage widget.

### How to test it?

- Go to the admin homepage and see what it looks like. I made the widget children red so it's obvious what part is the widget and what part is the children. The children should be scrollable
- To test the error and loading states, comment the children and uncomment each one of them
